infra/dev/scripts/setup-postgres-linux.sh

```diff
@@ -53,8 +53,10 @@ TARGETARCH=$(dpkg --print-architecture)
 
 # Install PostgresSQL
 echo_header $GREEN "Step [1/4]: Installing PostgreSQL..."
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+wget -qO- https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo tee /etc/apt/trusted.gpg.d/pgdg.asc &>/dev/null
```

Comment:
@gitstart-twenty what is this doing?

Reply:
Hey @charlesBochet 
56: First we add the official PostgreSQL repository to the system's list of software sources.
57: We then download the GPG key used to sign the Postgres packages and add it to the list of trusted keys.